docker/Makefile

```diff
@@ -1,16 +1,15 @@
 NAMESPACE ?= puppet
-git_describe := $(shell git describe)
+git_describe = $(shell git describe)
 vcs_ref := $(shell git rev-parse HEAD)
 build_date := $(shell date -u +%FT%T)
 hadolint_available := $(shell hadolint --help > /dev/null 2>&1; echo $$?)
 hadolint_command := hadolint --ignore DL3008 --ignore DL3018 --ignore DL3028 --ignore DL4000 --ignore DL4001
 hadolint_container := hadolint/hadolint:latest
-pwd := $(shell pwd)
-export BUNDLE_PATH = $(pwd)/.bundle/gems
-export BUNDLE_BIN = $(pwd)/.bundle/bin
-export GEMFILE = $(pwd)/Gemfile
+export BUNDLE_PATH = $(PWD)/.bundle/gems
+export BUNDLE_BIN = $(PWD)/.bundle/bin
+export GEMFILE = $(PWD)/Gemfile
 
-version := $(shell echo $(git_describe) | sed 's/-.*//')
+version ?= $(shell echo $(git_describe) | sed 's/-.*//')
```

Comment:
Why '?='  ?
Is overriding this useful?

Reply:
When working on development for other containers I've used the ability to override `version` fairly regularly. So, not sure it's needed here, but it's at least a bit more consistent. `:=` is wrong anyways, and that was failing in CI (since you need to un-shallow the clone before you can `git describe`), so it would need to be `=`.

Reply:
Cool. Was mostly curious about the '?'; agree that the := would definitely be problematic